packages/principal/test/account.spec.js

```diff
@@ -0,0 +1,17 @@
+import { Account } from '../src/lib.js'
+import { assert } from 'chai'
+
+export const utf8 = new TextEncoder()
+describe('Account', async () => {
+  it('it can sign', async () => {
+    const account = Account.from({ id: 'did:mailto:alice@web.mail' })
```

Comment:
nit: should this test using the `did:mailto` syntax that disallows unescaped `@`?